diff -urp /home/luca/devel/Cross/local/i586-pc-msdosdjgpp/include/stddef.h include/stddef.h
--- /home/luca/devel/Cross/local/i586-pc-msdosdjgpp/include/stddef.h	Fri May  3 17:31:10 2002
+++ include/stddef.h	Thu Jan  6 23:23:47 2005
@@ -18,6 +18,9 @@ __DJ_size_t
 __DJ_wchar_t
 #undef __DJ_wchar_t
 #define __DJ_wchar_t
+__DJ_wint_t
+#undef __DJ_wint_t
+#define __DJ_wint_t
 
 #ifndef __dj_ENFORCE_ANSI_FREESTANDING

diff -u -d -r newlib-1.13.0/newlib/libc/include/sys/types.h freedos-32/apps/newlib/include/sys/types.h
--- newlib-1.13.0/newlib/libc/include/sys/types.h	Thu Jul  3 04:32:08 2003
+++ freedos-32/apps/newlib/include/sys/types.h	Tue Apr 26 07:10:18 2005
@@ -157,7 +157,10 @@
 #endif
 
 #ifndef __CYGWIN__	/* which defines these types in it's own types.h. */
-typedef long		off_t;
+#ifndef	_OFF_T_
+#define	_OFF_T_
+typedef long long int off_t;
+#endif	/* Not _OFF_T_ */
 
 typedef	unsigned short	uid_t;
 typedef	unsigned short	gid_t;
